tyr/servers/nginx/server.py

```diff
@@ -35,5 +38,19 @@ def __init__(self, group=None, server_type=None, instance_type=None,
                                           chef_path, subnet_id, dns_zones)
 
     def configure(self):
-
+        super(NginxServer, self).establish_logger()
         super(NginxServer, self).configure()
+
+    @property
+    def user_data(self):    
+        # read in userdata file
+        self.log.info("Loading user-data [user_data_file.ps2]")
```

Comment:
This should probably be moved to something "autoscaling Nginx"-specific that doesn't use `ps2` as the extension.